manifest.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "ideacrm",
+  "vendor": "vtex",
+  "version": "0.0.0",
+  "description": "Idea CRM pixel app",
+  "builders": {
+    "pixel": "0.x"
+  },
+  "dependencies": {
+    "vtex.pixel-interfaces": "1.x"
+  },
+  "credentialType": "absolute",
+  "settingsSchema": {
+    "title": "Idea CRM Options",
+    "type": "object",
+    "properties": {
+      "idc": {
+        "title": "Idea CRM ID",
+        "description": "Enter your Idea CRM ID",
+        "type": "string"
+      },
+      "pre_empresa": {
+        "title": "Idea CRM Company",
+        "description": "Enter your Idea CRM Company",
+        "type": "string"
+      },
+      "pre_depto": {
+        "title": "Idea CRM Departament",
+        "description": "Enter your Idea CRM Departament",
+        "type": "string"
+      },
+      "floatingchat": {
+        "title": "Enable Idea CRM floating chat",
+        "type": "boolean"
```

Comment:
I think it would be better if some defaults were set

To set a default value just use:

```json
  "default": true
```